### ISSUE TITLE
Shuffling and Video Player Type

### ISSUE THREAD

**Issue by Liquoriced** [jellyfin/jellyfin-android#332]
When shuffling TV episodes, Android client does not use external player

1. Go to 'TV Shows'
2. Click on 'three dots' on any TV Series
3. Click on 'Shuffle'
4. Jellyfin utilizes internal video player, even though external video player (i.e., VLC) is selected in the options and is used when choosing directly the episode to play

**Expected behavior**
When choosing an external video player in the 'Client Settings', the external video player is only used when selecting directly the video to play. The external video player should also be used when watching TV Shows in Shuffle mode.

**System (please complete the following information):**
 - App version: 2.2.3
 - Android version: Android 11
 - Device manufacturer and model: Samsung Galaxy S10
 - Jellyfin server version: 10.7.1
 - The type of video player implementation used: external video player (VLC)

**Comment by Liquoriced** [comment]
Same issue is present in v2.3.0-beta.1.

**Comment by nielsvanvelzen** [comment]
Can't reproduce in the latest beta.

**Comment by Liquoriced** [comment]
I just checked again, and the issue persists with the v2.3.0-beta.1.

Below is the log from playing an episode by selecting it (Jellyfin Android uses VLC), and by using the shuffle function (Jellyfin Android uses internal player).
I noticed that when I use the shuffle function the log lists an Unknown Profile, even though in the Client Settings the player is still set to external (see **normal play**, below).
Could this be the issue?


**NORMAL PLAY**
[2021-05-11 10:00:29.841 +08:00] [INF] [45] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Android External Player", Path: "\\192.168.1.101\video\Tvshows\The Big Bang Theory (2007)\The.Big.Bang.Theory.S12.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-EPSiLON (68.33GB)\The.Big.Bang.Theory.S12E22.The.Maternal.Conclusion.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-EPSiLON.mkv", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-05-11 10:00:29.842 +08:00] [INF] [45] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Android External Player", Path: "\\192.168.1.101\video\Tvshows\The Big Bang Theory (2007)\The.Big.Bang.Theory.S12.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-EPSiLON (68.33GB)\The.Big.Bang.Theory.S12E22.The.Maternal.Conclusion.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-EPSiLON.mkv", isEligibleForDirectPlay: True, isEligibleForDirectStream: True

**SHUFFLE PLAY**
[2021-05-11 10:00:19.905 +08:00] [INF] [22] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Unknown Profile", Path: "\\192.168.1.101\video\Tvshows\The Big Bang Theory (2007)\The.Big.Bang.Theory.S12.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-EPSiLON (68.33GB)\The.Big.Bang.Theory.S12E24.The.Stockholm.Syndrome.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-EPSiLON.mkv", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-05-11 10:00:19.905 +08:00] [INF] [22] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Unknown Profile", No video direct play profiles found for "\\192.168.1.101\video\Tvshows\The Big Bang Theory (2007)\The.Big.Bang.Theory.S12.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-EPSiLON (68.33GB)\The.Big.Bang.Theory.S12E24.The.Stockholm.Syndrome.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-EPSiLON.mkv" with codec "h264"
[2021-05-11 10:00:19.906 +08:00] [INF] [22] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Unknown Profile", Path: "\\192.168.1.101\video\Tvshows\The Big Bang Theory (2007)\The.Big.Bang.Theory.S12.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-EPSiLON (68.33GB)\The.Big.Bang.Theory.S12E24.The.Stockholm.Syndrome.1080p.BluRay.REMUX.AVC.DTS-HD.MA.5.1-EPSiLON.mkv", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
